modules/__tests__/Media-test.js

```diff
@@ -133,7 +133,7 @@ describe('A <Media>', () => {
         )
 
         render(element, node, () => {
-          expect(node.firstChild.innerHTML).toNotMatch(/hello/)
+          expect(node.firstChild.innerHTML).not.toBeDefined()
```

Comment:
@SimenB If I remember correctly this might be a manual change I did, as `node.firstChild.innerHTML` is not defined AND Jest `.not.toMatch(/hello/)` fails in that case. Not sure if that should be considered a bug in Jest.